### DISCUSSION TITLE
Do Free Accounts have Account-Secrests (versus Repo only Secrets)?

### DISCUSSION THREAD

**Discussion by PureKrome** [community/community#25747] [Actions]
<p>:wave:t2: G’Day awesome people!</p>
<p>I have a free GH account (side note: I am part of some other, paid organizations though).</p>
<p>A lot of my OSS repo’s end up having the same secret(s) in them: for example, package management secrets like NuGet or NPM etc…</p>
<p>Is there a way for people to just enter this -once- and any of our repo’s can use/get access to these ‘common’ secret(s) ?</p>

**Comment by bill-richards** [comment, marked as answer]
<p>TLDR;</p>
<p>Not for a free personal account.<br>
Yes for a free organisation account</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/actions/security-guides/encrypted-secrets">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/actions/security-guides/encrypted-secrets" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/43810257/181093014-d2d30f94-d822-4e2a-9d64-c7bf836fc2d7.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/actions/security-guides/encrypted-secrets" target="_blank" rel="noopener nofollow ugc">Encrypted secrets - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/43810257/181093015-1c635443-fbbb-451a-a01a-60f6dad01f57.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations" target="_blank" rel="noopener nofollow ugc">About organizations - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Personally, I would be questioning <em><strong>the validity of the secrets</strong></em> which are shared across multiple projects. They cease to be secrets worth having at that point.</p>

**Comment by PureKrome** [comment]
<p><a class="mention" href="/bill-richards">@bill-richards</a> thanks heaps Bill for the answers and links - really appreciate it.</p>

**Comment by PureKrome** [comment]
<p>Oh! I forgot to ask → what about if we have a paid GH account? Or is it more linked to Orgs than Personal Accounts?</p>

**Comment by bill-richards** [comment]
<aside class="quote no-group" data-username="PureKrome" data-post="4" data-topic="220716">
<div class="title">
<div class="quote-controls"></div>
 PureKrome:</div>
<blockquote>
<p>what about if we have a paid GH account? Or is it more linked to Orgs than Personal Accounts?</p>
</blockquote>
</aside>
<p>Read the links that I so helpfully provided for you</p>

**Comment by PureKrome** [comment]
<p>Thanks <a class="mention" href="/bill-richards">@bill-richards</a> ! I’ll <em>re-read</em> them again.</p>
